jdbc-v2/src/main/java/com/clickhouse/jdbc/PreparedStatementImpl.java

```diff
@@ -532,6 +532,20 @@ private static String encodeObject(Object x) throws SQLException {
                     sb.append(new String(buffer, 0, len));
                 }
                 return "'" + escapeString(sb.toString()) + "'";
+            } else if (x instanceof Object[]) {
+                StringBuilder arrayString = new StringBuilder();
+                arrayString.append("[");
+                int i = 0;
+                for (Object item : (Object[]) x) {
+                    if (i > 0) {
+                        arrayString.append(", ");
```

Comment:
it will be an extra `,` anyway. 
So I would append it at the end and just truncate `arrayString`